traffic_portal/test/integration/PageObjects/DeliveryServicePage.po.ts

```diff
@@ -0,0 +1,212 @@
+        await this.txtOrgServerURL.sendKeys('http://origin.infra.ciab.test');
+        await this.txtProtocol.sendKeys('HTTP')
+        break;
+      }
+      case "STEERING": {
+        await this.txtXmlId.sendKeys(deliveryservice.Name + this.randomize);
+        await this.txtDisplayName.sendKeys(deliveryservice.Name + this.randomize);
+        await this.selectActive.sendKeys('Active')
+        await this.selectType.sendKeys('STEERING')
+        await this.selectTenant.sendKeys('- root')
+        await this.selectCDN.sendKeys('dummycdn')
+        await this.txtProtocol.sendKeys('HTTP')
+        break;
+      }
+      default:
+        {
+          console.log('Wrong Type name');
+          break;
+        }
+    }
+    await this.btnCreateDeliveryServices.click();
+    result = await basePage.GetOutputMessage().then(value => value === deliveryservice.validationMessage);
+    return result;
+  }
+  public async SearchDeliveryService(nameDS: string): Promise<boolean> {
+    let name = nameDS + this.randomize;
+    await this.txtSearch.clear();
+    await this.txtSearch.sendKeys(name);
+    const result = await element(by.cssContainingText("span", name)).isPresent();
+    await element(by.cssContainingText("span", name)).click();
+    return !result;
+  }
+  public async UpdateDeliveryService(deliveryservice: UpdateDeliveryService): Promise<boolean | undefined> {
+    let result: boolean | undefined = false;
+    let basePage = new BasePage();
+    if (deliveryservice.validationMessage.includes("updated")) {
+      deliveryservice.validationMessage = deliveryservice.validationMessage.replace(deliveryservice.Name, deliveryservice.Name + this.randomize)
+    }
+    switch (deliveryservice.description) {
+      case "update delivery service display name":
+        await this.txtDisplayName.clear();
+        await this.txtDisplayName.sendKeys(deliveryservice.NewName + this.randomize);
+        await basePage.ClickUpdate();
+        break;
+      default:
+        result = undefined;
+    }
+    if (result = !undefined) {
+      result = await basePage.GetOutputMessage().then(value => value === deliveryservice.validationMessage);
+    }
+    return result;
+  }
+  public async DeleteDeliveryService(deliveryservice: DeleteDeliveryService): Promise<boolean> {
+    let result = false;
+    let basePage = new BasePage();
+    if (deliveryservice.validationMessage.includes("deleted")) {
+      deliveryservice.validationMessage = deliveryservice.validationMessage.replace(deliveryservice.Name, deliveryservice.Name + this.randomize);
+    }
+    await this.btnDelete.click();
+    await this.txtConfirmName.sendKeys(deliveryservice.Name + this.randomize);
+    await basePage.ClickDeletePermanently();
+    result = await basePage.GetOutputMessage().then(value => value === deliveryservice.validationMessage);
+    return result;
+  }
+  public async AssignServerToDeliveryService(deliveryservice: AssignServer): Promise<boolean>{
```

Comment:
nit: Can we have a new line between functions? Easier to see where they end and new one starts